tests/e2e/deletion_test.go

```diff
@@ -71,7 +71,8 @@ func (tc *testContext) testApplicationDeletion(component components.ComponentInt
 
 	if err := wait.PollUntilContextTimeout(tc.ctx, tc.resourceRetryInterval, tc.resourceCreationTimeout, false, func(ctx context.Context) (bool, error) {
 		appList, err := tc.kubeClient.AppsV1().Deployments(tc.applicationsNamespace).List(ctx, metav1.ListOptions{
-			LabelSelector: labels.ODHAppPrefix + component.GetComponentName(),
+			//LabelSelector: labels.ODHAppPrefix + component.GetComponentName(),
+			LabelSelector: labels.ODH.Component(component.GetComponentName()),
```

Comment:
4 other places? Or it's just to check?

Reply:
pushed too fast............ not even finshed changes

Reply:
> pushed too fast............ not even finshed changes

Ah, sorry. Been there :)